src/main/java/frc/robot/subsystems/BallShooter.java

```diff
@@ -115,10 +115,10 @@ public void periodic() {
         SmartDashboard.putNumber("shooter/Motor Output", motorOutput);
         SmartDashboard.putNumber("shooter/Motor Velocity", shootMotor.getSelectedSensorVelocity(0));
         SmartDashboard.putNumber("shooter/Motor Error", shootMotor.getClosedLoopError(0));
-        SmartDashboard.putNumber("shooter/Motor Velocity(RPM)",
+        SmartDashboard.putNumber("DT/shooter/Motor Velocity(RPM)",
                 velocityPer100msToRpm(shootMotor.getSelectedSensorVelocity(0)));
 
-        SmartDashboard.putNumber("shooter/Shooter Temp", shootMotor.getTemperature());
+        SmartDashboard.putNumber("DT/shooter/Shooter Temp", shootMotor.getTemperature());
```

Comment:
You can create hierarchy like this, but I'm not sure it is what you were thinking.  I'll approve it, but lets talk about it next time we meet.